### DISCUSSION TITLE
Printing Templates

### DISCUSSION THREAD

**Discussion by Kirkc88** [directus/directus#17197] [Feature Requests]
Create PDF or Printable templates based on data from collections.

**Comment by paescuj** [comment]
What would be your use case? I can imagine that in most cases this would lead to PDFs with a lot of pages...
You could also export the data as CSV and then convert it to a PDF if needed.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
